build/nuke/Native/OpenALSoft.cs

```diff
@@ -0,0 +1,106 @@
+                    var build = $"cmake --build . --config Release{JobsArg}";
+
+                    if (OperatingSystem.IsWindows())
+                    {
+                        foreach (var (platform, rid) in new[]
+                        {
+                            ("Win32", "win-x86"),
+                            ("x64", "win-x64"),
+                            ("ARM64", "win-arm64"),
+                        })
+                        {
+                            EnsureCleanDirectory(buildDir);
+
+                            InheritedShell($"{prepare} -A {platform}", buildDir).AssertZeroExitCode();
+                            InheritedShell(build, buildDir).AssertZeroExitCode();
+
+                            CopyFile(buildDir / "Release" / "OpenAL32.dll", runtimes / rid / "native" / "soft_oal.dll", FileExistsPolicy.Overwrite);
+                        }
+                    }
+                    else if (OperatingSystem.IsLinux())
+                    {
+                        foreach (var (triple, rid) in new[]
+                        {
+                            ("x86_64-linux-gnu", "linux-x64"),
+                            ("arm-linux-gnueabihf", "linux-arm"),
+                            ("aarch64-linux-gnu", "linux-arm64"),
+                        })
+                        {
+                            EnsureCleanDirectory(buildDir);
+
+                            InheritedShell($"{prepare} {GetCMakeToolchainFlag(triple)}", buildDir).AssertZeroExitCode();
+                            InheritedShell(build, buildDir).AssertZeroExitCode();
+
+                            InheritedShell($"{triple}-strip --strip-unneeded libopenal.so", buildDir).AssertZeroExitCode();
+
+                            CopyFile(buildDir / "libopenal.so", runtimes / rid / "native" / "libopenal.so", FileExistsPolicy.Overwrite);
+                        }
+                    }
+                    else if (OperatingSystem.IsMacOS())
+                    {
+                        foreach (var (arch, rid) in new[]
+                        {
+                            ("x86_64", "osx-x64"),
+                            ("arm64", "osx-arm64"),
+                        })
+                        {
+                            EnsureCleanDirectory(buildDir);
+
+                            InheritedShell($"{prepare} -DCMAKE_OSX_ARCHITECTURES={arch}", buildDir).AssertZeroExitCode();
+                            InheritedShell(build, buildDir).AssertZeroExitCode();
+
+                            InheritedShell($"strip -Sx libopenal.dylib", buildDir).AssertZeroExitCode();
+
+                            CopyFile(buildDir / "libopenal.dylib", runtimes / rid / "native" / "libopenal.dylib", FileExistsPolicy.Overwrite);
+                        }
+                    }
+
+                    Git("checkout HEAD build/", OpenALSoftPath);
```

Comment:
Is this just to stop Git from being noisy in the CI logs?

Reply:
It's mostly for local testing. The submodule will show up as having changes in `git status` because the `.empty` file in that directory will have been removed.

Reply:
ok